pkgs/tools/filesystems/blobfuse/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, curl, gnutls, libgcrypt, libuuid, fuse }:
+
+stdenv.mkDerivation rec {
+  name = "blobfuse";
+  version = "1.0.1-RC-Preview";
+
+  src = fetchFromGitHub {
+    owner  = "Azure";
+    repo   = "azure-storage-fuse";
+    rev    = "v${version}";
+    sha256 = "143rxgfmprir4a7frrv8llkv61jxzq50w2v8wn32vx6gl6vci1zs";
+  };
+
+  buildInputs = [ cmake curl gnutls libgcrypt libuuid fuse ];
+  nativeBuildInputs = [ pkgconfig ];
+
+  meta = with stdenv.lib; {
+    description = "Mount an Azure Blob storage as filesystem through FUSE";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
Do you want to maintain this package?  https://nixos.wiki/wiki/Nixpkgs#Becoming_a_Nixpkgs_maintainer